share/picongpu/examples/TransitionRadiation/etc/picongpu/16.cfg

```diff
@@ -61,10 +61,6 @@ TBG_e_histogram="--e_energyHistogram.period 10 \
                  --e_energyHistogram.maxEnergy 500000 \
                  --e_energyHistogram.filter all"
 
-# optional hdf5 output
-TBG_hdf5="--hdf5.period 10\
-          --hdf5.file pos"
-
```

Comment:
Didn't replace with openPMD on purpose?

Reply:
well spotted.
This variable was not used by default in `TBG_plugins` I removed it that the user is not wondering why he had no output!

Reply:
Okay, that's good.